src/include/daos/tests_lib.h

```diff
@@ -28,6 +28,51 @@
 #include <daos_types.h>
 #include <daos/object.h>
 
+#define MIN_CHECK 16
+static inline void
+print_mismatch(const uint8_t *i1, const uint8_t *i2)
```

Comment:
this block is essentially to workaround a cmocka bug where it immediately exits when running a subset of tests....it spits the assertion to the daos log before it finishes.

Reply:
Just FYI, I tried https://github.com/daos-stack/daos/pull/420  and it can fix this cmocka bug.

Reply:
yeah, I'll probably remove this from the patch.  It was useful for debugging locally but if we have another means, that's probably better.   On the other hand, it would be useful to log just failures to daos log..I wonder if cmocka has any kind of hook to do that